Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN cargo build --release
 
 FROM debian:buster-slim
 
-RUN apt-get update && apt-get install -y libssl-dev
+RUN apt-get update && apt-get install -y libssl-dev ca-certificates
```

Comment:
You also have to run the command:

```console
$ update-ca-certificates --fresh
```

Reply:
I inserted it, just to be very sure, thanks:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/25957738/93878396-4b065e00-fcda-11ea-95cd-eaa13b229cc0.png">


<img width="378" alt="image" src="https://user-images.githubusercontent.com/25957738/93878313-28744500-fcda-11ea-8f72-2ce9c1227999.png">